ontotext-yasgui-web-component/README.md

```diff
@@ -83,6 +83,6 @@ The "config" value of "ngce-prop-config" or "[config]" is an object with followi
 - <b>showEditorTabs</b>: If the query editor tabs should be rendered or not;
 - <b>showResultTabs</b>: If the results tabs should be rendered or not;
 - <b>showToolbar</b>: If the toolbar with render mode buttons should be rendered or not.
-
+- <b>yasqePluginButtons</b>: Plugin definitions configurations for yasqe action buttons. 
```

Comment:
IMO. "..plugin.." is a little bit confusing. We have other functionality which use "plugin" term. Maybe "yasqeActionButtons" or just "yasqeActions" would be better. What you think?

Reply:
Maybe you are right. I'll think of another name. I just used the name of the yasqe config.